conda/recipe/install_libtvm.bat

```diff
@@ -15,7 +15,7 @@
 :: specific language governing permissions and limitations
 :: under the License.
 
-cmake --build build --config Release --target install
+cmake --build build --config Release --target install -- /p:CL_MPcount=8
```

Comment:
is it possible to use the windows equivalent of `nproc` here instead of always assuming it's `8`?

Reply:
I think =0 will use all available processors.  I didn't see a lot of improvement from this (plus googling suggests limiting this number), so I'll try 2 and 4 next.